api/v1alpha1/backendtrafficpolicy_types.go

```diff
@@ -98,6 +103,15 @@ type BackendTrafficPolicySpec struct {
 	Compression []*Compression `json:"compression,omitempty"`
 }
 
+type HTTPOptions struct {
```

Comment:
there are already some HTTP only fields at the top level like `FaultInjection`, so vote to keep this at a top level
will bring this up in the next envoy gateway community meeting

Reply:
OK. Will wait for the decision before changing this PR.